bayesian/api/api_v2.py

```diff
@@ -11,35 +11,85 @@
 from bayesian.exceptions import HTTPError
 from bayesian.utils import (get_system_version,
                             server_create_component_bookkeeping,
                             server_create_analysis,
                             check_for_accepted_ecosystem)
-import os
-from fabric8a_auth.errors import AuthError
 from bayesian.utility.v2.ca_response_builder import ComponentAnalyses
-from collections import namedtuple
-import logging
+from bayesian.utility.v2.sa_response_builder import (StackAnalysesResponseBuilder,
+                                                     SARBRequestInvalidException,
+                                                     SARBRequestInprogressException,
+                                                     SARBRequestTimeoutException)
+from bayesian.utility.v2.stack_analyses import StackAnalyses, SAInvalidInputException
+from bayesian.utility.v2.sa_models import StackAnalysesPostRequest
+from bayesian.utility.v2.backbone_server import BackboneServerException
+from bayesian.utility.db_gateway import RdbAnalyses, RDBSaveException, RDBInvalidRequestException
+
 
 errors = {
-        'AuthError': {
-                         'status': 401,
-                         'message': 'Authentication failed',
-                         'some_description': 'Authentication failed'
-                     }}
+    'AuthError': {
+         'status': 401,
+         'message': 'Authentication failed',
+         'some_description': 'Authentication failed'
+     },
+    'BackboneServerException': {
+        'status': 500,
+        'message': 'Error while reaching aggregator/recommender service',
+        'some_description': 'Exception raised by backbone server request'
+    },
+    'RDBSaveException': {
+        'status': 500,
+        'message': 'Error while saving request',
+        'some_description': 'Exception raised while saving data to RDB'
+    },
+    'RDBInvalidRequestException': {
+        'status': 400,
+        'message': 'Invalid request ID',
+        'some_description': 'Exception raised when request id is invalid'
+    },
+    'SARBRequestInvalidException': {
+        'status': 400,
+        'message': 'Worker result for request ID does not exist yet',
+        'some_description': 'Exception raised when result data is not fetch for a request id'
+    },
+    'SARBRequestInprogressException': {
+        'status': 202,
+        'message': 'Analysis for request ID is in progress',
+        'some_description': 'Exception raised when request id is in process'
+    },
+    'SARBRequestTimeoutException': {
+        'status': 408,
+        'message': 'Stack analysis request has timed out. Please retry with a new analysis',
+        'some_description': 'Exception raised when request id is timed out'
+    },
+    'SAInvalidInputException': {
+        'status': 400,
+        'message': 'Invalid dependencies encountered',
+        'some_description': 'Exception raised during parsing manifest file'
+    }
+}
```

Comment:
Using `Api(errors)` and Blueprint error handlers `api_v2.errorhandler(..)` seems to be redundant for me, Why do we need to have both in place?